src/app/components/List/EBookList.jsx

```diff
@@ -2,7 +2,20 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { Link } from 'react-router';
 import Dropdown from '../../libraries/react-dropdown-aria';
-import { flattenDeep, unique } from '../../util/Util';
+import { flattenDeep, unique, formatUrl } from '../../util/Util';
+
+/**
+ * Create a URL for that conforms to the webpub-reader streamed format
+ * @param {string} url
+ * @param {string} eReaderUrl
+ */
+const generateStreamedReaderUrl = (url, eReaderUrl) => {
+  const base64BookUrl = Buffer.from(formatUrl(url)).toString('base64');
+  const encodedBookUrl = encodeURIComponent(`${base64BookUrl}`);
+  const encodedReaderUrl = encodeURI(eReaderUrl);
+  const combined = `${eReaderUrl}/readerNYPL/?url=${encodedReaderUrl}/pub/${encodedBookUrl}/manifest.json`;
```

Comment:
How does this work with the `/manifest.json` at the end? I don't think its in the .epub files, right? It does work, I'm just curious what its doing.

Reply:
I think this is a result of the streamer, because what it's doing right now is taking the unexploded epubs and running it through the readium streamer so that it can be displayed to us.

Reply:
Okay, I think I understand? So its reading the generated `manifest.json` file, right? Or does it just expect that to be there and ignores it in what is actually being parsed?

Reply:
When the epub is streamed through the reader, it's made available through a link that's essentially [host]/[base_64_encoded_direct_link]/manifest.json 
So I don't think it'll be found if the manifest.json is stripped (like, url-wise, cuz it's not pointing to the right file anymore)

All we're doing is passing that url into RNow to show in an iframe.